api/src/resolvers/banking/banking-resolver.ts

```diff
@@ -26,6 +54,8 @@ const bankingMutation = {
 
     const { merchantId, auth } = getStreamFinancials(args.stream_name)
 
+    await checkIfLastServiceBanked(args.serviceRecordId, context)
+
     // This code checks if there has already been a successful transaction
     const transactionResponse = rearrangeCypherObject(
```

Comment:
I think we should check if the last offering was banked after checking if there has already been a successful transaction. 

It fees more efficient that way

Reply:
Ok. done